src/models/axis.js

```diff
@@ -89,7 +90,7 @@ nv.models.axis = function() {
                   };
                     axisLabel
                         .attr('text-anchor', 'middle')
-                        .attr('y', 0)
+                        .attr('y', -xLabelMargin)
```

Comment:
This changes the default to be -36 instead of 0, is that intentional?  I'm kind of wary of changing the default value on everyone.